### PR TITLE
test: add test case iot-qcow2-image

### PR DESCRIPTION
**Pull request by yih-redhat**

### CODE REVIEW

**Review by henrywang — CHANGES_REQUESTED** [review]

File: ostree-iot-qcow2.sh [hunk 1]
```diff
@@ -0,0 +1,637 @@
...
+        echo "unsupported distro: ${ID}-${VERSION_ID}"
+        exit 1;;
+esac
+
+# Colorful output.
+function greenprint {
+    echo -e "\033[1;32m${1}\033[0m"
+}
+
+# Compare rpm package version
```
[the remark this answers is not in the document]
Comment (yih-redhat) [reply ostree-iot-qcow2.sh]: fixed

File: ostree-iot-qcow2.sh [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+
+    # Start the compose.
+    greenprint "🚀 Starting compose"
+    if [ $# -eq 3 ]; then
+        repo_url=$3
+        sudo composer-cli --json compose start-ostree --ref "$OSTREE_REF" --url "$repo_url" "$blueprint_name" "$image_type" | tee "$COMPOSE_START"
+    elif [ $# -eq 4 ]; then
+        repo_url=$3
+        parent_ref=$4
+        sudo composer-cli --json compose start-ostree --ref "$OSTREE_REF" --parent "$parent_ref" --url "$repo_url" "$blueprint_name" "$image_type" | tee "$COMPOSE_START"
```
[the remark this answers is not in the document]
Comment (yih-redhat) [reply ostree-iot-qcow2.sh]: fixed

File: ostree-iot-qcow2.sh [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+sudo ostree --repo="$PROD_REPO" pull --mirror edge-stage "$OSTREE_REF"
+
+# Clean compose and blueprints.
+greenprint "🧽 Clean up container blueprint and compose"
+sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
+sudo composer-cli blueprints delete container > /dev/null
+
+############################################################
+##
+## Build edge-raw-image
```
[the remark this answers is not in the document]
Comment (yih-redhat) [reply ostree-iot-qcow2.sh]: fixed

**Comment by yih-redhat** [comment]
all comments resolved and test passed, ready for merge now.

**Comment by henrywang** [comment]
Thanks @yih-redhat. I merged PR #4990. Could you please make a rebase and use `/test-f38-qcow2` to trigger test.

**Review by henrywang — APPROVED** [review]

**Pull request merged**